network.bisq.Bisq.yml

```diff
@@ -0,0 +1,86 @@
+app-id: network.bisq.Bisq
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.openjdk17
+command: Bisq
+finish-args:
+    - "--share=network"
+    - "--share=ipc"
+    - "--socket=x11"
+    - "--socket=pulseaudio"
+    - "--device=dri"
+    - "--env=PATH=/usr/bin:/app/bin:/app/jdk/bin"
+    - "--env=JAVA_HOME=/app/jdk"
+modules:
+    - name: openjdk
+      buildsystem: simple
+      build-commands:
+          - /usr/lib/sdk/openjdk17/installjdk.sh
+    - name: bisq
+      buildsystem: simple
+      build-commands:
+          - gpg --import publickey.gpg; gpg --verify Bisq.deb.asc Bisq.deb
+          - ar x Bisq.deb
+          - rm Bisq.deb
+          - tar xf data.tar.xz
+          - cp -r opt/bisq/lib /app/lib
+          - install -D opt/bisq/bin/Bisq /app/bin/Bisq
+      sources:
+          - type: file
+            sha256: 310fa6381914d327ae1c1da995a78cad01df7f9a9b15863ad257f55194af9457
+            url: "https://bisq.network/downloads/v1.7.5/Bisq-64bit-1.7.5.deb"
+            dest-filename: Bisq.deb
+            only-arches:
+                - x86_64
+            x-checker-data:
+                type: anitya
+                project-id: 241531
+                url-template: "https://bisq.network/downloads/v$version/Bisq-64bit-$version.deb"
+          - type: file
+            url: https://bisq.network/downloads/v1.7.5/Bisq-64bit-1.7.5.deb.asc
+            dest-filename: Bisq.deb.asc
+            sha256: 84a634b4d5ad3d89c1130759f75cdb3e93c324b35538f8cdbd762ef6865ee929
+            only-arches:
+                - x86_64
+            x-checker-data:
+                type: anitya
+                project-id: 241531
+                url-template: https://bisq.network/downloads/v$version/Bisq-64bit-$version.deb.asc
+          - type: file
+            url: https://bisq.network/downloads/v1.7.5/29CDFD3B.asc
+            dest-filename: publickey.gpg
+            sha256: 61a87454943ab2229eb3f8b00e5a8558aa1ff7735af1f8f9bcf390d059832df3
+            x-checker-data:
+                type: anitya
+                project-id: 241531
+                url-template: https://bisq.network/downloads/v$version/29CDFD3B.asc
+    - name: icon
```

Comment:
usually icons, metainfo and desktop file are installed part of the main module. At least make them one module in addition to the "main" module. Anything is over kill.

Reply:
Merged